src/tours/Tour/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react';
+import Tour from 'reactour';
+import {
+    getGroupComparisonSteps,
+    getVirtualStudySteps,
+    groupComparisonId,
+    virtualStudyId,
+} from '../Steps';
+import { TourProps, TourMapProps } from './types';
+import './styles.scss';
+
+export const setTourLocalStorage = (id: string, value: string) => {
+    localStorage.setItem('web-tour', id);
+    localStorage.setItem(id, value);
+};
+
+export const setTourLocalStorageFromURL = () => {
+    /**
+     * If the url contains a query param 'webtour', set the localStorage
+     */
+    const urlStr = window.location.href.split('?')[1];
```

Comment:
we should obtain url params from routingStore in calling component.  that already has url parsed.